### ISSUE TITLE
Add a sense of 'height'. Rockers, aircraft, etc.

### ISSUE THREAD

**Issue by stefanhendriks** [stefanhendriks/Dune-II---The-Maker#151]
Ie, rockets from an ornithopter are fired 'from above' and should go 'down'. The illusion of height can be given by adding a shadow (which is always the 'real' position) and drawing the rocket 'above' it.

The same goes for the air units, the shadows are fixed now and should be more flexible. For carry-alls this will give more of an illusion to 'pick up' a unit by 'going down'.

## TODO
- [ ] make rockets go up when launched from ground unit & down when impacting (slight movement for small rockets, the deathhand has the most height as it has to go over the whole map)
- [ ] carry-all will go 'down' (near shadow) when picking up a unit and 'lift off' when bring that unit away

**Comment by arkahys** [comment]
I comment here, as it is really close to this.
To give better impress of height, draw air units at "zoom x1,1" when on "non-passable cell" and draw it at "zoom x0,9" when picking or enter/exit a structure.
Can be applied to rocket  when fired and before explose with i.e a degraded from 0,9 >1,1 > 0,9

**Comment by stefanhendriks** [comment]
yes, with the height/zoom we can play and give the impression that a carry-all is higher up (by rendering it a bit bigger).

So, lets say zoom 1.0 (normal) is on ground level. This is like normal units. But, when a carry-all would fly up, it can look a bit bigger. The shadow still is rendered at the correct cell (this is the actual position of the unit); the actual unit can be rendered a bit bigger when higher up. And when it goes down it will go back to zoom level 1. 

The zoom level playing is an addition, I'd like to start with shadows first @claude before doing anything with zooming.
